ddl/db_integration_test.go

```diff
@@ -1210,3 +1210,75 @@ func (s *testIntegrationSuite) TestAlterColumn(c *C) {
 	_, err = s.tk.Exec("alter table mc modify column a bigint auto_increment") // Adds auto_increment should throw error
 	c.Assert(err, NotNil)
 }
+
+func (s *testIntegrationSuite) assertWarningExec(c *C, sql string) {
```

Comment:
I think adding the warning message as an argument can be beneficial for calls to other functions.